tests/sentry/api/test_issue_search.py

```diff
@@ -218,6 +220,31 @@ def test_invalid(self):
             convert_query_values(filters, [self.project], self.user, None)
 
 
+@apply_feature_flag_on_cls("organizations:issue-states")
+class ConvertSubStatusValueTest(TestCase):
+    def test_valid(self):
+        for substatus_string, substatus_val in SUBSTATUS_UPDATE_CHOICES.items():
+            filters = [SearchFilter(SearchKey("substatus"), "=", SearchValue([substatus_string]))]
+            result = convert_query_values(filters, [self.project], self.user, None)
+            assert result[0].value.raw_value == [substatus_val]
+
+            filters = [SearchFilter(SearchKey("substatus"), "=", SearchValue([substatus_val]))]
+            result = convert_query_values(filters, [self.project], self.user, None)
+            assert result[0].value.raw_value == [substatus_val]
+
+    def test_invalid(self):
+        filters = [SearchFilter(SearchKey("substatus"), "=", SearchValue("wrong"))]
+        with pytest.raises(InvalidSearchQuery, match="invalid substatus value"):
+            convert_query_values(filters, [self.project], self.user, None)
+
+        filters = [AggregateFilter(AggregateKey("count_unique(user)"), ">", SearchValue("1"))]
+        with pytest.raises(
+            InvalidSearchQuery,
+            match=r"Aggregate filters \(count_unique\(user\)\) are not supported in issue searches.",
+        ):
+            convert_query_values(filters, [self.project], self.user, None)
```

Comment:
Maybe I'm missing something, but I'm not totally sure what this has to do with the changes in this PR.